src/Controller/Admin/AddressCrudController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Entity\Address;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Crud;
+use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
+use EasyCorp\Bundle\EasyAdminBundle\Field\DateTimeField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\FormField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\IdField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\NumberField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\TextField;
+
+class AddressCrudController extends AbstractCrudController
+{
+    public static function getEntityFqcn(): string
+    {
+        return Address::class;
+    }
+
+    public function configureCrud(Crud $crud): Crud
+    {
+        return $crud
+            ->setDefaultSort(['id' => 'DESC']);
```

Comment:
Consider using constant from https://github.com/doctrine/collections/blob/2.1.x/src/Criteria.php here, e.g. 

```suggestion
            ->setDefaultSort(['id' => Criteria::DESC]);
```

(cf. https://github.com/doctrine/collections/blob/2.1.x/src/Criteria.php#L22C1-L23)

Reply:
Fixed